packages/server/graphql/private/mutations/updateEmail.ts

```diff
@@ -15,6 +16,39 @@ const updateEmail: MutationResolvers['updateEmail'] = async (_source, {oldEmail,
     throw new Error(`User with ${oldEmail} not found`)
   }
 
+  // Update isEmailVerified to false for the old email
+  // Part 1: Get identities for the user from db
+  const currentIdentitiesQuery = await pg
+    .selectFrom('User')
+    .select('identities')
+    .where('email', '=', oldEmail)
+    .limit(1)
+    .executeTakeFirst()
```

Comment:
-1 We already load the whole `User` row in l14. I know we omit the `identity` row, but that's probably just to not need to have it typed out. I suggest replacing the `getUserByEmail` call with a call to kysely, something like
```ts
const user = await pg
  .selectFrom('User')
  .selectAll()
  .where('email', '=', oldEmail)
  .executeTakeFirst()
```

Reply:
Makes sense, this way we would not be making a call to the user table twice.